digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/LinearAssetService.scala

```diff
@@ -475,7 +496,7 @@ trait LinearAssetOperations {
 
       toInsert.foreach{ linearAsset =>
         val id = dao.createLinearAsset(linearAsset.typeId, linearAsset.linkId, linearAsset.expired, linearAsset.sideCode,
-          linearAsset.startMeasure, linearAsset.endMeasure, linearAsset.createdBy.getOrElse(LinearAssetTypes.VvhGenerated), linearAsset.vvhTimeStamp)
+          linearAsset.startMeasure, linearAsset.endMeasure, linearAsset.createdBy.getOrElse(LinearAssetTypes.VvhGenerated), linearAsset.vvhTimeStamp, getLinkSource(linearAsset.linkId))
```

Comment:
We already have the link geometry source in the roadlink sequence. So we don't need to be calling VVH again.